### PR TITLE
load from string or file

### PR DESCRIPTION
**Pull request by gleeper**
Updated the config to load yaml from a file (`init_from_file`) or a string (`init`).

Also updated tests to test both init paths.

### CODE REVIEW

**Comment by gleeper** [comment]
[Actual GCP Authorization Strategy](https://trello.com/c/gJH1b2nP/30-actual-gcp-authorization-strategy)

File: test/config_test.py [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
 from absl.testing import absltest
-from config_loader import config
 import jinja2
 import yaml
 
+from yaml_config_loader import config
+
 
 @contextlib.contextmanager
 def config_file(content):
```
Comment (jgan) [inline test/config_test.py:28]: I know it's not related to this PR, but I like this helper. I'm going to steal this for other stuff. :)

File: yaml_config_loader/config.py [hunk 2]
```diff
@@ -31,41 +31,32 @@ class _Config(object):
...
-    file_name = os.path.basename(file_name)
-
     # Load the template
-    jinja_loader = jinja2.FileSystemLoader(directory)
-    jinja_env = jinja2.Environment(
-        loader=jinja_loader, undefined=jinja2.StrictUndefined)
-    template = jinja_env.get_template(file_name)
+    template = jinja2.Environment(
+        loader=jinja2.BaseLoader,
+        undefined=jinja2.StrictUndefined).from_string(yaml_string)
```
Comment (jgan) [inline yaml_config_loader/config.py:57]: 👍for keeping template behavior on string

**Review by jgan — APPROVED** [review]
LGTM

**Pull request merged**